src/family/family.component.ts

```diff
@@ -184,8 +195,7 @@ export class FamilyComponent implements OnInit, OnDestroy {
         }
 
         this.locations = res.data;
-        this.countries = _
-          .chain(res.data)
+        this.countries = chain(res.data)
```

Comment:
this.countriesFilterServiceSubscribe = this.countriesFilterService
       .getCountries(`thing=${this.thing}`)
       .filter((res) => { 
         if (res.error) {
            console.error(res.err);
            return false;
         }
         return true;
       })
       .do((res) => {
         this.locations = res.data;
         return res.data;
       })
       .mergeMap(data => data.countries)
       .map(countries => _.sortBy('country'))
       .subscribe((sortedCountries: any) => this.countriesTranslations = sortedCountries);